### PR TITLE
Phantomcss visualtesting

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [x] Include a change request file if publishing 
- [x] New feature, bugfix, or enhancement
- [x] Documentation update

#### Description of changes

Add in phantomcss visual regression testing.

### CODE REVIEW

**Comment by micahgodbolt** [comment]
Looks like we'll need to supress those casper types not starting with an I.

File: packages/office-ui-fabric-react/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" user-scalable="no" />
 
-  <title>Office UI Fabric React Examples</title>
+  <title>testpage</title>
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/index.html]: I'm guessing we'll want to change this back?

Comment (joschect) [inline packages/office-ui-fabric-react/index.html]: There really isn't a reason to, this page isn't used.

**Comment by micahgodbolt** [comment]
@joschect should we close this PR down until we're close to a finished product, or do you still need it here for travis testing?

**Comment by joschect** [comment]
Sounds good. I'll close it.

**Pull request closed without merging**